powers.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+int main ()
```

Comment:
Nice job, but you didn't handle all inputs correctly! You also forgot to include 2^0 = 1.